bin/internal/update_dart_sdk.sh

```diff
@@ -55,18 +55,18 @@ if [ ! -f "$ENGINE_STAMP" ] || [ "$ENGINE_VERSION" != `cat "$ENGINE_STAMP"` ]; t
 
   curl --continue-at - --location --output "$DART_SDK_ZIP" "$DART_SDK_URL" 2>&1 || {
     echo
-    echo "Failed to retrieve the Dart SDK at $DART_SDK_URL"
-    echo "If you're located in China, please follow"
-    echo "https://github.com/flutter/flutter/wiki/Using-Flutter-in-China"
+    echo "Failed to retrieve the Dart SDK from: $DART_SDK_URL"
```

Comment:
The Windows version of this file `update_dart_sdk.ps1` should probably also include this warning. Some users in China may be on Windows.

Reply:
Looks like that script doesn't know how to determine if the download failed. I'll file a bug about adding that; I would rather my "change some strings" PR didn't turn into a "learn powershell" PR... :-)

Reply:
Somebody once told me "embrace the yak shave!" 😛

Reply:
I'm knee-deep in yak hair as it is. :-P